### PR TITLE
E2E Tests for App Upload

### PR DESCRIPTION
**Pull request by Yibaebi**

### CODE REVIEW

File: mirage/factories/sbom-file.ts [hunk 1]
```diff
@@ -2,18 +2,37 @@ import { Factory } from 'miragejs';
...
+
+  sb_project(i: number) {
+    return 1000 + i + 1;
+  },
+
+  completed_at: () => faker.date.recent().toString(),
+  created_at: () => faker.date.past().toString(),
+};
+
 export default Factory.extend({
```
Comment (future-pirate-king) [inline mirage/factories/sbom-file.ts:24]: above object should be used

File: mirage/factories/sbom-project.ts [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+  project(i: number) {
+    return i + 1;
+  },
+
+  latest_sb_file(i: number) {
+    return 100 + i + 1;
+  },
+};
+
 export default Factory.extend({
```
Comment (future-pirate-king) [inline mirage/factories/sbom-project.ts:17]: above object should be used

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: app/components/upload-app/status/index.hbs [hunk 3]
```diff
@@ -3,6 +3,7 @@
     <div local-class='divider'></div>
 
     <AkLoader
+      aria-label='upload-app-status-loader'
```
Comment (future-pirate-king) [inline app/components/upload-app/status/index.hbs:6]: can we have this in readable form ?

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: app/components/upload-app/status/index.hbs (hunk 3 above)
Comment (Yibaebi) [reply app/components/upload-app/status/index.hbs:6]: This isn't used. Will be removed.

**Review by future-pirate-king — APPROVED** [review]

**Pull request merged**